week3/src/Modal.tsx

```diff
@@ -49,7 +39,7 @@ const Container = styled.div`
   justify-content: center;
 `;
 
-const WhiteBox = styled.div`
+const WhiteBox = styled.div<{ correct?: boolean }>`
```

Comment:
이런방식으로 props를 전달해주는 방식이 정석인지 궁금해요!

Reply:
오옹 맞는데 옵셔널 체이닝 사용하지 않고도 가능해요!!! 
```suggestion ts
const WhiteBox = styled.div<{ correct: boolean }>`
 background-color: white;
  width: 300px;
  display: flex;
  flex-direction: column;
  align-items: center;
  z-index: 100;
  ${{correct} =>
    correct &&
    css`
      animation: smoothAppear 0.8s;
    `}
`
```
이렇게 하고
호출할때  `<WhiteBox correct={true}>`요러케! 해줍니다!

근데 옵셔널 잘 안써요!

Reply:
오옹 맞는데 옵셔널 체이닝 사용하지 않고도 가능해요!!! 
```suggestion ts
const WhiteBox = styled.div<{ correct: boolean }>`
 background-color: white;
  width: 300px;
  display: flex;
  flex-direction: column;
  align-items: center;
  z-index: 100;
  ${{correct} =>
    correct &&
    css`
      animation: smoothAppear 0.8s;
    `}
`
```
이렇게 하고
호출할때  `<WhiteBox correct={true}>`요러케! 해줍니다!

근데 옵셔널 잘 안써요!

Reply:
혹시 옵셔널을 안쓰는 이유가 해당 컴포넌트에서 직접 props를 넘겨주기 때문에 props가 없을 경우가 없어서,,,? 옵셔널 안쓰는 건가요?! 
안쓰는 이유가 궁금해요! ㅋㅋ